### PR TITLE
Add docstrings

### PR DESCRIPTION
**Pull request by nwithan8**
Most of our methods were documented, but some were missing docstrings and/or had formatting errors. Those have been corrected. Only public methods (those that an end-user would need to call) have been documented, which can be leveraged by doc generation or IDE code tips.

- Added docstrings to all public class methods.
- Fixed spelling errors
- Standardized verbiage and formatting

### CODE REVIEW

File: EasyPost/Client.cs [hunk 1]
```diff
@@ -1,91 +1,89 @@
 ﻿using Newtonsoft.Json;
```
Comment (Justintime50) [inline EasyPost/Client.cs:1]: You need to rebase so the user-agent changes aren't included in this PR.

File: EasyPost/Order.cs [hunk 2]
```diff
@@ -89,7 +89,7 @@ public void GetRates()
 
             rates = request.Execute<Order>().rates;
         }
-
+
```
Comment (Justintime50) [inline EasyPost/Order.cs:92]: Let's not include whitespaces.

File: EasyPost/Request.cs [hunk 3]
```diff
@@ -7,7 +7,7 @@
 namespace EasyPost {
     public class Request {
         internal RestRequest restRequest;
-
+
```
Comment (Justintime50) [inline EasyPost/Request.cs:10]: No extra whitespaces

**Review by Justintime50 — CHANGES_REQUESTED** [review]
Good stuff, just a couple extra random whitespaces that need removing and it needs to be rebased. Otherwise good.

File: EasyPost/Client.cs (hunk 1 above)
Comment (nwithan8) [reply EasyPost/Client.cs]: Patched in bc436ee

File: EasyPost/Order.cs (hunk 2 above)
Comment (nwithan8) [reply EasyPost/Order.cs]: Patched with 90a14e5

File: EasyPost/Request.cs (hunk 3 above)
Comment (nwithan8) [reply EasyPost/Request.cs]: Patched with 90a14e5

**Review by Justintime50 — APPROVED** [review]

**Review by jchen293 — APPROVED** [review]
lgtm

**Pull request merged**